### PR TITLE
chore(capman): add more tenant_ids to tests

### PR DESCRIPTION
**Pull request by volokluev**
Just updating some tests to be in line with snuba allocation policy requirements. no functional changes

Allows https://github.com/getsentry/snuba/pull/4098 to be merged

### CODE REVIEW

File: src/sentry/utils/snuba.py [hunk 1]
```diff
@@ -948,6 +947,8 @@ def _snql_query(params: Tuple[SnubaQuery, Hub, Mapping[str, str], str]) -> RawRe
     request, forward, reverse = query_data
     request.parent_api = parent_api
     assert isinstance(request, Request)
+    assert "organization_id" in request.tenant_ids
+    assert "referrer" in request.tenant_ids
```
Comment (volokluev) [inline src/sentry/utils/snuba.py:951]: shouldn't merge without reverting this

File: src/sentry/utils/snuba.py [hunk 2]
```diff
@@ -848,7 +847,7 @@ def _apply_cache_and_build_results(
 
     # Sort so that we get the results back in the original param list order
     results.sort()
-    # Drop the sort order val
+    # Drop the sort order valtests/snuba/api/endpoints/test_organization_event_details.py
```
Comment (barkbarkimashark) [inline src/sentry/utils/snuba.py:850]: ?

File: src/sentry/tsdb/snuba.py [hunk 3]
```diff
@@ -760,7 +760,7 @@ def get_range(
         return {k: sorted(result[k].items()) for k in result}
 
     def get_distinct_counts_series(
```
Comment (barkbarkimashark) [inline src/sentry/tsdb/snuba.py:762]: Could we update the base class method signatures as well: src/sentry/tsdb/base.py

**Review by barkbarkimashark — APPROVED** [review]
This looks mostly fine to me.

File: src/sentry/tsdb/snuba.py (hunk 3 above)
Comment (volokluev) [reply src/sentry/tsdb/snuba.py:762]: good call

**Pull request merged**